partner_time_to_pay/models/res_partner.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright 2022 - Moduon
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    d2p_life = fields.Float(
+        compute="_compute_d2x",
+        string="AVG Days to Payable (lifetime)",
+        store=True,
+    )
+    d2p_ytd = fields.Float(
+        compute="_compute_d2x",
+        string="AVG Days to Payable (YTD)",
+        store=True,
+    )
+    d2r_life = fields.Float(
+        compute="_compute_d2x",
+        string="AVG Days to Receivable (lifetime)",
+        store=True,
+    )
+    d2r_ytd = fields.Float(
+        compute="_compute_d2x",
+        string="AVG Days to Receivable (YTD)",
+        store=True,
+    )
```

Comment:
Please add `help` on these fields. The names use some acronyms that might not be obvious at first sight.